tests/util/fake_statsd_server.py

```diff
@@ -64,6 +66,19 @@ def _run_server(self):
 
             sock = socket.socket(socket.AF_UNIX, socket.SOCK_DGRAM)
             sock.settimeout(3)
+
+            # Increase the receiving buffer size where needed (e.g. MacOS has 4k RX
+            # buffers which is half of the max packet size that the client will send.
+            if os.name != 'nt':
+                recv_buff_size = sock.getsockopt(socket.SOL_SOCKET, socket.SO_RCVBUF)
+                if recv_buff_size <= self.MIN_RECV_BUFFER_SIZE:
+                    sock.setsockopt(
+                        socket.SOL_SOCKET,
+                        socket.SO_RCVBUF,
+                        self.MIN_RECV_BUFFER_SIZE,
```

Comment:
should it not be `recv_buff_size` ?

Reply:
Not here. `recv_buff_size` holds the current value of the buffer size and the `self.MIN_RECV_BUFFER_SIZE` is the target we want to set it to.